supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -45,6 +45,17 @@
 
 *See also*:
 
+[discrete]
+[[ACK]]
+==== image:images/yes.png[yes] ACK flag (noun)
+*Description*: The acknowledgement flag ("ACK flag") in a TCP packet indicates that the acknowledgement field in the packet is relevant. Do not use "ACK" as acronym for "acknowledgement" in general.
```

Comment:
Also, this second sentence sounds like it doesn't necessarily belong in this entry, but might instead belong as its own entry set to "Use it: no". wdyt?